apps/main/src/loan/hooks/useLoanPositionDetails.ts

```diff
@@ -97,8 +106,12 @@ export const useLoanPositionDetails = ({
       loading: userLoanDetailsLoading ?? true,
     },
     borrowAPY: {
-      value: loanDetails?.parameters?.rate ? Number(loanDetails?.parameters?.rate) : null,
-      thirtyDayAvgRate: thirtyDayAvgRate,
+      rate: loanDetails?.parameters?.rate ? Number(loanDetails?.parameters?.rate) : null,
+      // TODO: add rebasing yield when available in the snapshots endpoint
```

Comment:
Should we move this TODO to the prices API implementation? We can already have a temporary field there